extensions/azurecore/src/azurecore.d.ts

```diff
@@ -491,6 +494,7 @@ declare module 'azurecore' {
 			fullName: string;
 			defaultDatabaseName: string;
 		}
+
```

Comment:
nit: unnecessary space (remove from sqlops-protocol too if needed)

Reply:
I'll let it stay, there's no space as of now.